lib/factory-helper/timestamps.rb

```diff
@@ -0,0 +1,31 @@
+module FactoryHelper
+  class Timestamps < Base
+
+    class << self
+      def between(from= ::Time.at(0), to= ::Time.local(2050, 12, 31,  23,  23,  59, 999_999.999))
```

Comment:
Tangential to this PR, but nevertheless - [food for thought on spacing around `=`](https://gist.github.com/NikitaAvvakumov/38bd16152207365b6812#file-method_vs_operator-rb)